specification/deploymentmanager/resource-manager/Microsoft.DeploymentManager/preview/2019-11-01-preview/deploymentmanager.json

```diff
@@ -0,0 +1,2465 @@
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/serviceTopologyName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServiceTopologyResource"
+            }
+          },
+          "default": {
+            "description": "All non-success conditions are handled using the standard error contract.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ServiceTopologies"
+        ],
+        "operationId": "ServiceTopologies_Delete",
+        "summary": "Deletes the service topology.",
+        "x-ms-examples": {
+          "Delete topology": {
+            "$ref": "./examples/servicetopology_delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/serviceTopologyName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The topology was successfully deleted."
+          },
+          "204": {
+            "description": "The topology was not found and the operation is considered success."
+          },
+          "default": {
+            "description": "All non-success conditions are handled using the standard error contract.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DeploymentManager/serviceTopologies": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/subscriptionId"
+        },
+        {
+          "$ref": "#/parameters/resourceGroupName"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "ServiceTopologies"
+        ],
+        "operationId": "ServiceTopologies_List",
+        "summary": "Lists the service topologies in the resource group.",
+        "x-ms-examples": {
+          "List topologies": {
+            "$ref": "./examples/servicetopologies_list.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
is this list pagination result?

Reply:
No, we plan to enhance this with a paginated response in the future.